src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -801,14 +820,12 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
               'a bug in React. Please file an issue.',
           );
           // We just completed a root. Commit it now.
-          priorityContext = TaskPriority;
```

Comment:
Why is this not needed anymore?

Reply:
Rather than change the priority context every time we start on a new bucket, and then change it to task in the commit phase, and then reset it to the previous value when an error is thrown, I found it was simpler to move this complexity to `getPriorityContext`. https://github.com/acdlite/react/blob/867aeac5819c997bfee0a0ddf7d973624ef5e3d2/src/renderers/shared/fiber/ReactFiberScheduler.js#L1520-L1552